packages/contractkit/src/consts.ts

```diff
@@ -0,0 +1 @@
+export const gasInflationFactor = 1.3
```

Comment:
Nit: I think usually we capitalize constants like these

Reply:
agree

Reply:
Done.